src/scripts.js

```diff
@@ -90,18 +90,21 @@ const setTripsDestination = () => {
   let destinationObj = destinationRepo.findDestination(trip.destination);
   trip.destination = destinationObj;
   });
-}
+};
 
 const findYearlySpent = (travelersTrips) => {
   let yearlySpent = 0;
   travelersTrips.forEach(trip => {
     if(trip.date.includes('2022')) {
```

Comment:
Is there a more accurate and dynamic way you could find out if a trip happened "in the last year".

For example: will this function work as intended in 2023?

Look into [new Date()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date)

Reply:
I have it hard coded at the moment, definitely not dynamic.. but I have to do more research for newDate()

Reply:
Cool. Lets come back to this later!